pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -1,39 +1,36 @@
 :ruby
   hoc_mode = DCDO.get("hoc_mode", CDO.default_hoc_mode)
-  hoc_cse_enable_svg = DCDO.get("hoc_cse_enable_svg", false)
   show_single_hero = Homepage.show_single_hero(request)
   heroes_arranged, hero_display_time = Homepage.get_heroes_arranged(request)
   actions = Homepage.get_actions(request)
 
 :css
-  .cse-hero-message-line2 {
+  .csc-hero-message-line2 {
     font-size: 52px;
   }
 
   @media screen and (min-width: 576px) and (max-width: 900px) {
-    .cse-hero-message-line2 {
+    .csc-hero-message-line2 {
       font-size: 38px;
     }
   }
 
-  #cse-main-actions {
-    box-shadow: 0px 0px 20px 20px rgb(255 255 255), 0px 0px 20px 20px rgb(255 255 255);
+  #pageheader-wrapper #pageheader {
+    margin: 0 !important;
   }
 
-  @media screen and (max-width: 970px) {
-    #cse-main-actions {
-      box-shadow: 0px 0px 20px 10px rgb(255 255 255), 0px 0px 20px 10px rgb(255 255 255);
-    }
-  }
+- csc_colors = [{r: 137, g: 83, b: 161}, {r: 137, g: 200, b: 119}, {r: 60, g: 134, b: 199}, {r: 249, g: 163, b: 76}, {r: 140, g: 126, b: 187}, {r: 232, g: 80, b: 155}, {r: 88, g: 108, b: 179}, {r: 210, g: 138, b: 188}, {r: 137, g: 200, b: 119}, {r: 100, g: 196, b: 169}]
```

Comment:
what does this line do? It doesn't look like this value gets used.

Reply:
Oh, that was for some potential text-colouring which we've decided not to do for now.  Removing.